plugins/modules/provision_workflow_manager.py

```diff
@@ -1281,22 +1351,23 @@ def provision_wireless_device(self):
 
         site_exist, site_id = self.get_site_id(site_name)
 
-        primary_ap_location_site_id_list = []
-        secondary_ap_location_site_id_list = []
+        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") <= 0:
+            primary_ap_location_site_id_list = []
+            secondary_ap_location_site_id_list = []
 
-        if primary_ap_location:
-            self.log("Processing primary access point locations", "INFO")
-            for primary_sites in primary_ap_location:
-                self.log("Retrieving site ID for primary location: {0}".format(primary_ap_location), "DEBUG")
-                site_exist, primary_ap_location_site_id = self.get_site_id(primary_sites)
-                primary_ap_location_site_id_list.append(primary_ap_location_site_id)
+            if primary_ap_location:
+                self.log("Processing primary access point locations", "INFO")
+                for primary_sites in primary_ap_location:
+                    self.log("Retrieving site ID for primary location: {0}".format(primary_ap_location), "DEBUG")
```

Comment:
Hope you wanted to print the primary_sites instead of primary_ap_location

`        self.log("Retrieving site ID for primary location: {0}".format(primary_sites), "DEBUG")
`